### DISCUSSION TITLE
doris最新版1.2.2安装过程中，启动start_be.sh报错，不知道如何解决？！

### DISCUSSION THREAD

**Discussion by xiaohangtx** [apache/doris#17695] [A - General / Q&A] [answered]
doris最新版1.2.2安装过程中，启动start_be.sh报错，不知道如何解决？！

![image](https://user-images.githubusercontent.com/106477642/224500850-8400d831-7ca2-4bc2-9392-8428244d2127.png)

**Comment by LemonLiTree** [comment, marked as answer]
java udf jar包放在be\lib 目录下
